node-module-system/a-wrapped-module.js

```diff
@@ -0,0 +1,11 @@
+
+console.log(`type of exports: ${typeof exports}`);
+console.log(`value of exports: ${exports}`);
+console.log(`type of require: ${typeof require}`);
+console.log(`value of require: ${require}`);
+console.log(`type of module: ${typeof module}`);
+console.log(`value of module: ${module}`);
```

Comment:
In general, it'd be a good idea to pass objects as separate arguments when `log`ging them out so you can see their contents, rather than converting them to strings. However, for this exercise, it's fine.